ethportal-api/src/types/content_key/state.rs

```diff
@@ -1,140 +1,154 @@
-use crate::{types::content_key::overlay::OverlayContentKey, utils::bytes::hex_encode_compact};
-use ethereum_types::{U256, U512};
+use crate::{
+    types::{content_key::overlay::OverlayContentKey, state_trie::nibbles::Nibbles},
+    utils::bytes::hex_encode_compact,
+    ContentKeyError,
+};
+use ethereum_types::{Address, H256};
 use sha2::{Digest as Sha2Digest, Sha256};
-use sha3::{Digest, Keccak256};
-use ssz::{Decode, Encode};
+use ssz::{Decode, DecodeError, Encode};
 use ssz_derive::{Decode, Encode};
-use ssz_types::{typenum, FixedVector, VariableList};
 use std::fmt;
 
+// TODO(morph): replace with correct link once PR is merged.
```

Comment:
Typically we prefer opening issues & assigning to self, trying to avoid todo's scattered around the codebase

Reply:
Done. I don't have permission to assign it to myself or set labels. Can you please do it?